src/buffer/out/TextColor.cpp

```diff
@@ -82,6 +82,11 @@ bool TextColor::IsDefault() const noexcept
     return _meta == ColorType::IsDefault;
 }
 
+bool TextColor::IsDefaultOrIndex16() const noexcept
+{
+    return _meta != ColorType::IsRgb && _index < 16;

```

Comment:
`ColorType::IsDefault` has a `_index` of 0 at all times as mandated by the constructor.